matrix-meetings-bot/src/service/MeetingService.ts

```diff
@@ -606,6 +606,14 @@ export class MeetingService {
       }
 
       promises.push(this.cleanupWidgets(widgetIds, room, userContext));
+      const filteredWidgetIds = widgetIds.filter(
+        (widget) =>
+          !widget.includes(WidgetType.COCKPIT) &&
+          !widget.includes(WidgetType.BREAKOUT_SESSIONS)
```

Comment:
Bot creates cockpit and breakout widget ids in the form: `${WidgetType.COCKPIT}-${WidgetClient.createUUID()}`. I suggest to use `startsWith` to follow the format more explicit.